src/RCall.jl

```diff
@@ -29,13 +30,24 @@ export RObject,
    robject, rcopy, rparse, rprint, reval, rcall, rlang,
    rimport, @rimport, @rlibrary, @rput, @rget, @var_str, @R_str
 
-const depfile = joinpath(dirname(@__FILE__),"..","deps","deps.jl")
-if isfile(depfile)
-    include(depfile)
+@static if @has_preference("Rhome") || @has_preference("libR")
+    @static if !(@has_preference("Rhome") && @has_preference("libR"))
+        error("RCall: Either both Rhome and libR must be set or neither of them")
```

Comment:
not in this PR, but I would like to see the ability to figure out well-behaved libR when only Rhome is set

Reply:
I guess having a default would be fine, but yes please, let's defer it for now.